docs/editing.md

```diff
@@ -0,0 +1,65 @@
+# Editing manual sources
+
+## Introduction
+Before making any changes to the manual - either the English version or a
+translation, make sure you have read the [style guidelines](style.md)!
+
+## Editing existing documentation
+Simply open the files and edit them.
+
+## Adding new documentation
+When adding new functions or methods, there are a couple of options.
+
+### Option A: Generating files using docgen
+This is preferred way to generate files for new extensions, classes, functions
+or methods using [`docgen`][docgen]. The script is found in the [doc-base][doc-base]
+repository and uses reflection to generate documentation (DocBook) files.
+
+### Option B: Copy skeleton files
+This involves copying the skeleton files into the correct location:
+```
+cp doc-base/skeletons/method.xml classname/methodname.xml   #for new methods
+cp doc-base/skeletons/function.xml functions/functionname.xml #for new functions
+```
+
+Note: *classname*, *methodname* and *functionname* are lowercased names of the
+class, method, or function, respectively, not a literal file name.
```

Comment:
We should mention `gen_stub.php` from the php-src repo. As that is the better way even if it lacks support for some aspects.